Source/santactl/Commands/SNTCommandMetrics.m

```diff
@@ -0,0 +1,181 @@
+  NSDictionary *normalizedMetrics = [self normalize:metrics];
+
+  if ([arguments containsObject:@"--json"]) {
+    // Format
+    NSData *metricData = [NSJSONSerialization dataWithJSONObject:normalizedMetrics
+                                                         options:NSJSONWritingPrettyPrinted
+                                                           error:nil];
+    NSString *metricStr = [[NSString alloc] initWithData:metricData encoding:NSUTF8StringEncoding];
+    printf("%s\n", [metricStr UTF8String]);
+  } else {
+    if (exportMetrics) {
+      printf(">>> Metrics Info\n");
+      printf("  %-25s | %s\n", "Metrics Server", [metricsURLStr.absoluteString UTF8String]);
+      printf("  %-25s | %s\n", "Metrics Format",
+             [[self stringFromMetricFormat:metricFormat] UTF8String]);
+      printf("  %-25s | %lu\n", "Export Interval (seconds)", metricExportInterval);
+      printf("\n");
+    } else {
+      printf("Metrics not configured\n");
+      return;
+    }
+
+    printf(">>> Root Labels\n");
+
+    for (NSString *label in normalizedMetrics[@"root_labels"]) {
+      const char *labelStr = [label cStringUsingEncoding:NSUTF8StringEncoding];
+      const char *valueStr =
+        [normalizedMetrics[@"root_labels"][label] cStringUsingEncoding:NSUTF8StringEncoding];
+
+      printf("  %-25s | %s\n", labelStr, valueStr);
+    }
+
+    printf("\n");
+    printf(">>> Metrics \n");
+
+    for (NSString *metricName in normalizedMetrics[@"metrics"]) {
+      NSDictionary *metric = normalizedMetrics[@"metrics"][metricName];
+      const char *metricNameStr = [metricName UTF8String];
+      const char *description = [metric[@"description"] UTF8String];
+      NSString *metricType = SNTMetricMakeStringFromMetricType([metric[@"type"] integerValue]);
+      const char *metricTypeStr = [metricType UTF8String];
+
+      printf("  %-25s | %s\n", "Metric Name", metricNameStr);
+      printf("  %-25s | %s\n", "Description", description);
+      printf("  %-25s | %s\n", "Type", metricTypeStr);
+
+      for (NSString *fieldName in metric[@"fields"]) {
+        for (NSDictionary *field in metric[@"fields"][fieldName]) {
+          const char *fieldNameStr = [fieldName cStringUsingEncoding:NSUTF8StringEncoding];
+          const char *fieldValueStr = [field[@"value"] cStringUsingEncoding:NSUTF8StringEncoding];
+          const char *createdStr = [field[@"created"] UTF8String];
+          const char *lastUpdatedStr = [field[@"last_updated"] UTF8String];
+          const char *data = [[NSString stringWithFormat:@"%@", field[@"data"]] UTF8String];
+
+          if (strlen(fieldNameStr) > 0) {
+            printf("  %-25s | %s=%s\n", "Field", fieldNameStr, fieldValueStr);
+          }
+
+          printf("  %-25s | %s\n", "Created", createdStr);
+          printf("  %-25s | %s\n", "Last Updated", lastUpdatedStr);
+          printf("  %-25s | %s\n", "Data", data);
```

Comment:
nit: I'd 'label' this Value rather than Data.

Reply:
so we're using the `Value` key as the field value and data as the contents in `SNTMetricSet` so I was trying to be consistent with the existing code.